File: app/src/main/java/com/zillennium/utswap/screens/setting/kycScreen/selfieHoldingScreen/SelfieHoldingActivity.java
package com.zillennium.utswap.screens.setting.kycScreen.selfieHoldingScreen;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.LinearLayout;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.zillennium.utswap.bases.BasePassedActivity;
import com.zillennium.utswap.databinding.ActivityKycSelfieHoldingBinding;
import com.zillennium.utswap.screens.setting.kycScreen.SelfieActivity;
import com.zillennium.utswap.screens.setting.kycScreen.verificationScreen.EmploymentInfo;

public class SelfieHoldingActivity extends AppCompatActivity {

    ActivityKycSelfieHoldingBinding binding;

    @Nullable
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        binding = ActivityKycSelfieHoldingBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());


        LinearLayout btnCameraSelfie = binding.btnCameraSelfie;
        btnCameraSelfie.setOnClickListener(view -> {
            Intent intent = new Intent(getApplicationContext(), SelfieActivity.class);
            startActivity(intent);
        });

        LinearLayout btnNext = binding.btnNext;
        btnNext.setOnClickListener(view -> {
            Intent intent = new Intent(this, EmploymentInfo.class);
            startActivity(intent);
        });

        // Set Passed Back
        ImageView ivBack = binding.ivBack;
        BasePassedActivity.onPassedBack(SelfieHoldingActivity.this, ivBack);
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/kyc/selfieHoldingScreen/SelfieHoldingActivity.kt
package com.zillennium.utswap.screens.kyc.selfieHoldingScreen

import android.content.Intent
import com.zillennium.utswap.R
import com.zillennium.utswap.UTSwapApp
import com.zillennium.utswap.bases.mvp.BaseMvpActivity
import com.zillennium.utswap.databinding.ActivityKycSelfieHoldingBinding
import com.zillennium.utswap.screens.kyc.employmentInfoScreen.EmploymentInfoActivity
import com.zillennium.utswap.screens.kyc.idTypeScreen.camera.selfieCameraActivity.SelfieCameraActivity
import java.io.IOException

class SelfieHoldingActivity :
    BaseMvpActivity<SelfieHoldingView.View, SelfieHoldingView.Presenter, ActivityKycSelfieHoldingBinding>(),
    SelfieHoldingView.View {

    override var mPresenter: SelfieHoldingView.Presenter = SelfieHoldingPresenter()
    override val layoutResource: Int = R.layout.activity_kyc_selfie_holding

    override fun initView() {
        super.initView()
        try {
            binding.apply {

                btnCameraSelfie.setOnClickListener {
                    val intent = Intent(UTSwapApp.instance, SelfieCameraActivity::class.java)
                    startActivity(intent)
                }

                btnNext.setOnClickListener {
                    val intent = Intent(UTSwapApp.instance, EmploymentInfoActivity::class.java)
                    startActivity(intent)
                }

                ivBack.setOnClickListener {
                    onBackPressed()
                }
            }

        } catch (error: Exception) {
            // Must be safe
        }
    }
}